internal/internal_worker.go

```diff
@@ -821,7 +839,10 @@ func (aw *aggregatedWorker) Start() error {
 		}
 	}
 
-	aw.logger.Info("Started Worker")
+	if workerStarted {
```

Comment:
Remove this log. Instead log after workflowWorker.Start() and activityWorker.Start() individually.

Reply:
done